src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -332,13 +337,61 @@ public override IntermediateNode Visit(TexlCallNode node, IRTranslatorContext co
                     }
                 }
 
+                args = ReplaceBlankWithDefault(args, func);
+
                 // this can rewrite the entire call node to any intermediate node.
                 // e.g. For Boolean(true), Instead of IR as Call(Boolean, true) it can be rewritten directly to emit true.
                 var irNode = func.CreateIRCallNode(node, context, args, scope);
 
                 return MaybeInjectCoercion(node, irNode, context);
             }
 
+            private List<IntermediateNode> ReplaceBlankWithDefault(List<IntermediateNode> args, TexlFunction func)
+            {
+                var len = args.Count;
+                List<IntermediateNode> convertedArgs = new List<IntermediateNode>(len);
+                IntermediateNode convertedNode;
+                BlankHandlerPolicy blankHandlerPolicy;
+
+                for (var i = 0; i < len; i++)
+                {
+                    convertedNode = args[i];
```

Comment:
> convertedNode = args[i]; [](http://example.com/codeflow?start=19&length=25)

nit: move to 'default case. 

Unecessary assignment in the non-default cases.

Reply:
updated.